config/v1/types_image.go

```diff
@@ -22,7 +22,7 @@ type Image struct {
 }
 
 type ImageSpec struct {
-	// AllowedRegistriesForImport limits the container image registries that normal users may import
+	// allowedRegistriesForImport limits the container image registries that normal users may import
```

Comment:
@bparees lowercase is our convention (so it aligns with the YAML/json fields)? Usually this triggers a warning from golang linters.

/lgtm

Reply:
yeah, so it aligns w/ what the user would actually have to type in yaml.